### DISCUSSION TITLE
npm install @openzeppelin/contracts

### DISCUSSION THREAD

**Discussion by 0xkumar** [Cyfrin/foundry-full-course-cu#440] [Q&A]
This is the error I encountered when I run the command 

![Screenshot 2023-07-25 121656](https://github.com/Cyfrin/foundry-full-course-f23/assets/136639853/f170c215-4cab-49c9-a12f-96c48e1bbf05)

I have used the AI tools to rectify this is says 

![Screenshot 2023-07-25 121920](https://github.com/Cyfrin/foundry-full-course-f23/assets/136639853/d8db9160-2fbe-4d76-bcf0-6354d85efb94)

It suggested to try this commands and I have tried to run the commands but none of them worked

![Screenshot 2023-07-25 122124](https://github.com/Cyfrin/foundry-full-course-f23/assets/136639853/c8904e3f-81b8-4c27-8fc1-b4383b6bc7df)

**Comment by Naman1729** [comment]
Hey @0xkumar , Run this command :
```
forge install openzeppelin/openzeppelin-contracts@v4.9.2 --no-commit 
```

**Reply by 0xkumar** [reply]
yeah, this command works I know but the above command I mentioned will create a seperate npm_modules  folder.So i just want to try that but its not working for me.

**Reply by Naman1729** [reply]
Hey @0xkumar , I notice that the spelling of the contracts in command is wrong and I run this command it work for me :
```
   npm install @openzeppelin/contracts
```

**Reply by 0xkumar** [reply]
Hey @Naman1729, I'm using WSL. I think that might be the reason for this error.

**Comment by 0xkumar** [comment]
![Screenshot 2023-07-25 160803](https://github.com/Cyfrin/foundry-full-course-f23/assets/136639853/b9467dff-c944-4e98-9664-ad8ce5dbb4d5)
